_layouts/minimal.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: table_wrappers
+---
+
+<!DOCTYPE html>
+
+<html lang="{{ site.lang | default: 'en-US' }}">
+{% include head.html %}
+<body>
+  <a class="skip-to-main" href="#main-content">Skip to main content</a>
+  {% include icons/icons.html %}
+
```

Comment:
Gotcha - I won't add this to this batch right now, let me think about the best way forward is for this!

Reply:
I'll create a new issue from this!

Reply:
@kevinlin1 wrote:
> The unfortunate thing is that it also requires additional work to change `just-the-docs.js` so that the search bar is not hidden under the hamburger menu.

Maybe `just-the-docs.js` (and `head.html`) should also be split into components needed for specific features?